src/styles/index.scss

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  $minutes-diraction: 3600;
+  $seconds-diraction: 60s;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted black;
+  border-radius: 50%;
+
+  &--speed-up {
+    $minutes-diraction: 600s;
+    $seconds-diraction: 10s;
```

Comment:
Inside the modifier `&--speed-up`, the variables are redeclared with a unit 's' which is not necessary. Also, correct the typo in the variable names to `$minutes-duration` and `$seconds-duration`.